sdk/synapse/Microsoft.Azure.Management.Synapse/src/Generated/Operations.cs

```diff
@@ -701,13 +672,15 @@ public async Task<AzureOperationResponse<OperationResource>> GetAzureAsyncHeader
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "operationId");
             }
+            string apiVersion = "2021-06-01";
```

Comment:
Are you mixing different versions of API (e.g. `2021-06-01` and `2021-06-01-preview`)?

Reply:
Never mind. I found the answer.